kpi/deployment_backends/kc_access/storage.py

```diff
@@ -40,6 +40,12 @@ def __init__(
 
 class KobocatS3Boto3Storage(S3Boto3Storage):
 
-    def __init__(self, acl=None, bucket=None, **settings):
+    def __init__(self, *args, **kwargs):
         bucket = django_settings.KOBOCAT_AWS_STORAGE_BUCKET_NAME
-        super().__init__(acl=None, bucket=bucket, **settings)
+        super().__init__(*args, **kwargs)
+
+
+class KobocatAzureStorage(AzureStorage):
+    def __init__(self, *args, **kwargs):
+        azure_container = django_settings.KOBOCAT_AZURE_CONTAINER
```

Comment:
🤔  Same thoughts as above?  How does KobocatAzureStorage know when to write since `azure_container` is not passed to the parent constructor?

Reply:
Should not be something like this instead?:
```python
    def __init__(self, *args, **kwargs):
        kwargs['azure_container'] = django_settings. KOBOCAT_AZURE_CONTAINER
        super().__init__(*args, **kwargs)
```